blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -86,6 +87,9 @@ public HttpMetricsIngestionServer(ScheduleContext context, IMetricsWriter writer
         router.post("/v2.0/:tenantId/ingest/multi", new HttpMultitenantMetricsIngestionHandler(processor, timeout));
         router.post("/v2.0/:tenantId/ingest", new HttpMetricsIngestionHandler(processor, timeout));
         router.post("/v2.0/:tenantId/ingest/aggregated", new HttpStatsDIngestionHandler(processor, timeout));
+        EventModuleLoader.loadEventModule();
```

Comment:
There is no need to explicitly call `loadEventModule` over here as well...FIX FIX FIX!